param/version.py

```diff
@@ -177,6 +177,10 @@ def __init__(self, release=None, fpath=None, commit=None, reponame=None,
         """
         self.fpath = fpath
         self._expected_commit = commit
+
+        if release is not None or 'commit_count' in kwargs:
+            print('WARNING: param.Version now supports PEP440 and a new tag based workflow. See param/version.py for more details')
+
```

Comment:
I would like to use ``param.warning`` but that will probably cause cyclic import issues unless I inline the import? Is a print statement ok in this case and do you have any suggestions for a better message?

Reply:
Using print is ok by me.

Reply:
This is so simple I didn't test it locally. I would wait for travis here and also on autover where I'll port this change across now. The only thing I'm wondering about is whether this print will be triggered in some of the unit tests.